packages/mrm/src/index.js

```diff
@@ -113,6 +113,20 @@ function runAlias(aliasName, directories, options, argv) {
 	});
 }
 
+/**
+ * Returns the correct `mrm-` prefixed package name
+ *
+ * @param {"task" | "preset"} type
+ * @param {string} packageName
+ * @returns {string}
+ */
+function getResolvableName(type, packageName) {
```

Comment:
I'm not sure what "resolvable" means here. Wouldn't something like this more clear?

```suggestion
function getPackageName(type, packageName) {
```

Reply:
Changed to `getPackageName` and updated the tests :D